src/sage/combinat/posets/hasse_cython.pyx

```diff
@@ -30,8 +30,8 @@ class IncreasingChains(RecursivelyEnumeratedSet_forest):
     - ``positions`` -- a list of sets of integers describing the poset,
       as given by the lazy attribute ``_leq_storage`` of Hasse diagrams.
 
-    - ``element_constructor`` -- used to determine the type of chains,
-      for example ``list`` or ``tuple``
+    - ``constructor`` -- used to determine the type of chains,
+      for example :class:`list` or :class:`tuple`
```

Comment:
Can this be renamed without deprecation for the old name?

Reply:
this is not a named argument, just an argument

Reply:
All arguments are named arguments unless you use `, *,`. Users can do `D = IncreasingChains([{0,1},{1}], element_constructor=list, exclude=[]); D`

Reply:
We have `rename_keyword` but only for keywords, not for args. If you insist, I can make an ad-hoc warning, but I would say it's ok not to deprecate. In particular, it's not in the global namespace.

Reply:
maybe the name of the parameter can just be left as is?

Reply:
ok, done